app/views/accessibility_statement.scala.html

```diff
@@ -57,7 +57,7 @@ <h2 class="heading-large">@messages("accessibilityStatement.howAccessible.headin
             </ul>
 
             <h2 class="heading-large">@messages("accessibilityStatement.reportingProblems.heading")</h2>
-            <p>@Html(messages("accessibilityStatement.reportingProblems.para1", <a href="mailto:hmrc-accessibility-problems@digital.hmrc.gov.uk">{messages("accessibilityStatement.reportingProblems.para1.linkText")}</a>))</p>
+            <p>@Html(messages("accessibilityStatement.reportingProblems.para1", <a href={appConfig.accessibilityReportUrl(pageUri)}>{messages("accessibilityStatement.reportingProblems.para1.linkText")}</a>))</p>
```

Comment:
This doesn't get encoded when it comes through to the help form. It seems to work okay, but not sure if that would cause problems downstream. Plat-UI requested it encoded so I'd imagine there is a reason.